packages/amplify-graphql-searchable-transformer/src/cdk/create-es-domain.ts

```diff
@@ -10,7 +10,7 @@ export const createEsDomain = (stack: Construct, parameterMap: Map<string, CfnPa
   const { HasEnvironmentParameter } = ResourceConstants.CONDITIONS;
 
   const domain = new Domain(stack, ElasticsearchDomainLogicalID, {
-    version: ElasticsearchVersion.V6_2,
+    version: ElasticsearchVersion.V7_7,
```

Comment:
We want to update to `V7_10`. Is this just a limitation of the version of the CDK we're using? If so, we can look into other workarounds or even updating the version of CDK (updating would likely need to be done across the entire project).

Reply:
Apparently you would have to use a newer version of CDK to make that possible, which might not be trivial 

You can assign manual overrides on CDK
```js
 (domain.node.defaultChild as CfnDomain).elasticsearchVersion = "7.10"
```

Also go can you go through this if there is a path forward for upgrading to 7.10 without interruption